ckanext/dgu/theme/templates/package/resource_read.html

```diff
@@ -1,288 +1,163 @@
-      ${data_api_help(c.datastore_api)}
-    </py:if>
-
-    ${data_viewer_embed_dialog()}
+    {{data_viewer_embed_dialog()}}
 
     <py:if test="c.resource.get('description')">
       <div class="resource-notes" property="rdfs:label">
-        ${c.resource.get('description')}
+        {{c.resource.get('description')}}
       </div>
     </py:if>
 
     <div class="recline-read-only resource-preview">
-      <div class="preview-header" py:if="not h.is_wms(c.resource)">
+    {% if not h.is_wms(c.resource)%}
+      <div class="preview-header">
         <h2>Preview</h2>
         <p>Preview is currently available for files such as CSV, spreadsheets and plain text. <span id="ckanext-datapreview-source"></span></p>
       </div>
+      {% endif %}
       <div id="ckanext-datapreview"></div>
       <div id="ckanext-datapreview-extra"></div>
       <div id="ckanext-html-preview"></div>
     </div>
 
     <table class="table table-bordered">
-      <tr py:if="c.resource['last_modified']">
+      {% if c.resource['last_modified'] %}
+      <tr>
         <td class="key">Last updated:</td>
-        <td class="value">${h.render_datestamp(c.resource['last_modified'])}</td>
+        <td class="value">{{h.render_datestamp(c.resource['last_modified'])}}</td>
       </tr>
-      <tr py:with="fmt = (c.resource.get('format') or 'not specified').strip().lower()">
+      {% endif %}
+      {% with %}
+      {% set fmt = (c.resource.get('format') or 'not specified').strip().lower() %}
+      <tr>
         <td class="key">Format:</td>
-        <td class="value">${format_box(fmt)}</td>
+        <td class="value">{{m.format_box(fmt)}}</td>
       </tr>
-      <tr py:if="c.resource.get('resource_type') in ('file', None)">
+      {% endwith %}
+      {% if c.resource.get('resource_type') in ('file', None) %}
+      <tr>
         <!--! This is data, so show 5 stars of openness -->
         <td class="key">Resource Openness:</td>
-        <td class="value">${h.render_resource_stars(c.resource['id'])}</td>
+        <td class="value">{{h.render_resource_stars(c.resource['id'])}}</td>
       </tr>
+      {% endif %}
       <tr>
         <td class="key">Quality Check:<a name="qualitycheck"></a></td>
-        <td class="value">${h.render_qa_info_for_resource(c.resource)}</td>
+        <td class="value">{{h.render_qa_info_for_resource(c.resource)}}</td>
       </tr>
-      <tr py:for="field_dict, label_attributes, value_attributes in h.get_resource_fields(c.resource,dict(c.pkg_extras))">
-        <td class="dataset-label" py:attrs="label_attributes">${field_dict.get('label') or field_dict['name']}</td>
-        <td class="dataset-details" py:attrs="value_attributes">${field_dict.get('value') or 'No value'}</td>
+      {% for field_dict, label_attributes, value_attributes in h.get_resource_fields(c.resource, h.as_dict(c.pkg_extras)) %}
+      <tr>
+        <td class="dataset-label" py:attrs="label_attributes">{{field_dict.get('label') or field_dict['name']}}</td>
```

Comment:
`py:attrs="label_attributes"` needs converting. `label_attributes` is a dict here. I think you can just put `{{label_attributes|xmlattr}}`